File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/commands/CommandWatcherCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.arena.commands;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class CommandWatcherCreatorHub extends CreatorHub {
    public ArenaCommandWatcher createCommandWatcher(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (ArenaCommandWatcher) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/commands/CommandWatcherCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.arena.commands

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class CommandWatcherCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): ArenaCommandWatcher {
        return super.create(itemID, dataProvider) as ArenaCommandWatcher
    }
}